src/main/java/org/opensrp/repository/postgres/mapper/custom/xml/CustomLocationMapper.xml

```diff
@@ -23,10 +26,13 @@
 	      <if test="json != null">
 	        #{json,jdbcType=OTHER,typeHandler=org.opensrp.repository.postgres.handler.LocationTypeHandler},
 	      </if>
-	      nextVal('core.location_server_version_seq')
+	      nextVal('core.location_server_version_seq'),
+			<if test="geometry != null">
+				ST_GeomFromGeoJSON('{"type":"MultiPolygon","coordinates":[102.70283890000002,12.099235999999967]}')
```

Comment:
HI @githengi, I tried to test the function with the hardcoded JSON but it still did not work. Though, it works fine in my local environment. 
![doesnotexists](https://user-images.githubusercontent.com/62061579/105462411-685f2f00-5cb0-11eb-8309-6db2d78389a6.png)

Persistence of record in local database' snapshot is attached.
Can you please share your thoughts on this?